c/fibonacci/fibonacci.c

```diff
@@ -5,5 +5,5 @@
 int fib(int n) { return (n==0 || n==1) ? 1 : (fib(n-1) + fib(n-2)); }
 
 int main(int argc, char *argv[]) {
-  printf("Fibonacci: %d\n", fib((int) *argv[1] - '0'));
+  printf("Fibonacci Number: %d\n", fib((int) *argv[1] - '0'));
```

Comment:
It seems like @renanreboredo authored this, I'd like his input about approving this change before I merge the pull. Otherwise, feel free to submit a new PR or un-commit these changes.

Reply:
That's fine for me :wink: :+1:

Reply:
Thanks for the quick reply!